### ISSUE TITLE
[MAS4.1.2] Name property does not properly define for 'Phone No' on 'Show Restaurant information using  Adaptive card' button's content. 

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1361]
**Actual Result:**
When focus lands on the phone no., Narrator is only reading number "(206) 838-7388" which appearing on the screen.
Name property does not properly define for  'Phone no'.

**Expected Result:**
Name property should be properly define for  'Phone no'.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/ ) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Show Restaurant information using  Adaptive card " button by using Tab key and press Enter on it.
5. New data will upload.
6. Open Narrator (Ctrl + Win. Key+ Enter key)
7. Navigate to Phone No. "(206) 838-7388" by using Tab key.
8. Observe that Name for Phone No. "(206) 838-7388" is not defined properly.

**User Impact:**
If Name,State and Role property is not defined properly for controls then the users will get confuse about the purpose of that control.

**MAS Reference:**
[MAS4.1.2 - Name,Role,Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[MAS4.1.2_Name for Ph no..zip](https://github.com/Microsoft/BotFramework-WebChat/files/2584589/MAS4.1.2_Name.for.Ph.no.zip)

**Comment by compulim** [comment]
Reopening so we can confirm from the accessibility team.

**Comment by corinagum** [comment]
Confirmed with accessibility team as an external issue

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
